jupyter_fsspec/handlers.py

```diff
@@ -251,6 +251,77 @@ def post(self):
         self.finish()
 
 
+# ====================================================================================
+# Handle Syncing Local and Remote filesystems
+# ====================================================================================
+class FilesystemSyncHandler(BaseFileSystemHandler):
+    def initialize(self, fs_manager):
+        self.fs_manager = fs_manager
+
+    async def get(self):
+        # remote to local (fetching latest changes)
+        request_data = json.loads(self.request.body.decode("utf-8"))
+        local_destination_path = request_data.get("local_path")
```

Comment:
Were we ever thinking of using pydantic? Unwrapping all the keys and validating seems a little tedious. 
The endpoint doc must document verbosely what the expected input is like (e.g., this might eventually appear in swagger generated docs).